ns/rdfs.ttl

```diff
@@ -14,7 +14,8 @@ rdfs:Resource a rdfs:Class ;
 rdfs:Proposition a rdfs:Class ;
 	rdfs:isDefinedBy <http://www.w3.org/2000/01/rdf-schema#> ;
 	rdfs:label "Proposition" ;
-	rdfs:comment "An atomic, logical, abstract proposition." .
+	rdfs:comment "The class of abstract, atomic, logical propositions." ;
```

Comment:
I would prefer to reuse the [definition given in RDF-Concepts](https://www.w3.org/TR/rdf12-concepts/#dfn-proposition), which I find clearer.

```suggestion
	rdfs:comment "The class propositions, simple logical expressions, describing a relationship between two entities." ;
```